docs/pages/tutorial/layout.md

```diff
@@ -0,0 +1,467 @@
+  return (
+    <View style={styles.container}>
+      <ImageViewer placeholderImageSource={PlaceholderImage} />
+      <StatusBar style="auto" />
+    </View>
+  );
+}
+
+/* @hide const styles = StyleSheet.create({ */
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#25292e',
+    alignItems: 'center',
+  },
+});
+/* @end */
+```
+
+## Step 6: Create buttons using Pressable
+
+A modern mobile device typically constitutes a touch screen as the primary interface between an application and the user. Any level of event interaction in an application can be achieved by using a button that, on a physical device, is a simple finger touch.
+
+React Native provides various components to handle touch events on native platforms. For this tutorial, let's use the [`Pressable`](https://reactnative.dev/docs/pressable). It is a core component wrapper that can detect various stages of interactions, from basic single tap events to advance events such as the long press. It can be customized in terms of style and appearance, and you can extend the functionality by creating a custom wrapper component around it.
+
+From the layout, there are two buttons you have to implement. Each has different styles and labels. Let's start by creating a function component that can be reused to create the two buttons.
+
+- Create a new file called **Button.js** inside the **components** folder.
+- Then, import the `Pressable` component from React Native
+- Create a custom component called `Button` with the following code snippet:
+
+<!-- prettier-ignore -->
+```js
+import { StyleSheet, View, Pressable, Text } from 'react-native';
+
+export default function Button({ label }) {
+  return (
+    <View style={styles.buttonContainer}>
+      <Pressable style={styles.button} onPress={() => alert('You pressed a button.')}>
+        <Text style={styles.buttonLabel}>{label}</Text>
+      </Pressable>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  buttonContainer: {
+    width: 320,
+    height: 68,
+    marginHorizontal: 20,
+    alignItems: 'center',
+    justifyContent: 'center',
+    padding: 3,
+  },
+  button: {
+    borderRadius: 10,
+    width: '100%',
+    height: '100%',
+    alignItems: 'center',
+    justifyContent: 'center',
+    flexDirection: 'row',
+  },
+  buttonIcon: {
+    paddingRight: 8,
+  },
+  buttonLabel: {
+    color: '#fff',
+    fontSize: 16,
+  },
+});
+```
+
+The `Button` component receives a `label` [prop](https://reactnative.dev/docs/intro-react#props) so that anytime this component is used, the button can have a text displayed.
+
+Now, in the app, when the user will touch any of the buttons on the screen, a dialog box will be displayed. This is done by using an `alert()` function to display a static message. Currently, it is used as a placeholder since both buttons do not trigger any functionality right now.
+
+Now let's import this component into **App.js** and see it in action:
```

Comment:
```suggestion
Let's import this component into **App.js** and see it in action:
```
Since the previous paragraph started with "Now", let's remove this one